examples/koa/src/client.ts

```diff
@@ -1,29 +1,55 @@
-'use strict';
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-import { setupTracing } from "./tracer";
-const tracer = setupTracing('example-koa-client');
 import * as api from '@opentelemetry/api';
-import { default as axios } from 'axios';
+import * as axios from 'axios';
+// eslint-disable-next-line import/extensions
+import { setupTracing } from './tracer';
```

Comment:
FYI: This need to `eslint-disable-next-line import/extensions` will be dropped once *all* examples/ packages are using the top-level eslint.config.js and we can delete examples/.eslintrc. It is the `"extends": "airbnb-base",` in "examples/.eslintrc" that is making this necessary for now.

(The alternative is to delete "examples/.eslintrc" and possibly break `npm run lint:examples` in the top-level until I complete all the items in https://github.com/open-telemetry/opentelemetry-js-contrib/issues/2891)